src/plugins/graph/components/attribute-label.tsx

```diff
@@ -52,10 +52,10 @@ export const AttributeLabel = observer(
     }, [dataConfiguration, graphModel.plotType, place]);
 
     const getLabel = useCallback(() => {
-      if (defaultAxisLabels?.[place]) {
-        return defaultAxisLabels[place];
-      }
       if (useClickHereCue) {
```

Comment:
The only thing that makes me wonder if this is the best fix is this new arrangement may make developers think the default axis labels should be clickable. In CLUE, they're not even though `useClickHereCue` is `true` when CLUE uses the default axis labels. Maybe that's not a significant enough concern, though?